src/main/java/com/eternalcode/lobbyheads/head/sound/SoundController.java

```diff
@@ -0,0 +1,41 @@
+package com.eternalcode.lobbyheads.head.sound;
+
+import com.eternalcode.lobbyheads.configuration.implementation.HeadsConfiguration;
+import com.eternalcode.lobbyheads.head.event.HeadUpdateEvent;
+import com.eternalcode.lobbyheads.position.Position;
+import com.eternalcode.lobbyheads.position.PositionAdapter;
+import org.bukkit.Location;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+import java.util.UUID;
+
+public class SoundController implements Listener {
```

Comment:
to samo co w Particle controllerze

Reply:
Okay, i want opinion from another team members.